src/Symfony/Component/Mailer/composer.json

```diff
@@ -18,6 +18,7 @@
     "require": {
         "php": ">=7.2.5",
         "egulias/email-validator": "^2.1.10|^3",
+        "psr/event-dispatcher": "^1",
         "psr/log": "^1|^2|^3",
         "symfony/deprecation-contracts": "^2.1",
         "symfony/event-dispatcher": "^4.4|^5.0|^6.0",
```

Comment:
Shouldn't we remove `symfony/event-dispatcher` from the requirements?

Reply:
See my note about this in the description. 🙃